commit 585a2fd6a8c28cb3c5e8e64c20a892250994f58b
Author: Radek Novacek <rnovacek@redhat.com>
Date:   Tue May 5 14:51:15 2015 +0200

    Add a note that changing hypervisor_id will result in duplicated entries

diff --git a/virt-who-config.5 b/virt-who-config.5
index 3d8f883..5ff599d 100644
--- a/virt-who-config.5
+++ b/virt-who-config.5
@@ -44,7 +44,7 @@ Only hosts which uuid is specified in comma-separated list in this option will b
 Hosts which uuid is specified in comma-separated list in this option will \fBNOT\fR be reported. Put the uuid into the double-quotes if it contains special characters (like comma).
 .TP
 \fBhypervisor_id\fR
-Property that should be used as identification of the hypervisor. Can be one of following: \fBuuid\fR, \fBhostname\fR, \fBhwuuid\fR. Note that some virtualization backends don't have all of them implemented. Default is \fBuuid\fR. Applicable to esx, hyperv and rhevm only.
+Property that should be used as identification of the hypervisor. Can be one of following: \fBuuid\fR, \fBhostname\fR, \fBhwuuid\fR. Note that some virtualization backends don't have all of them implemented. Default is \fBuuid\fR. Applicable to esx, hyperv and rhevm only. This property is meant to be set up before initial run of virt-who. Changing it later will result in duplicated entries in the subscription manager.
 
 .SH EXAMPLE
 [test-esx]
